Cargo.toml

```diff
@@ -11,6 +11,8 @@ path = "src/main.rs"
 
 [workspace]
 members = [
+    "bin/bench",
```

Comment:
Would it be possible to use these bench and testing crates from the Substrate repo?

Reply:
Nope, the things that can be used from substrate repo are being used directly but these 3 packages required changes to be made. About 20% of these is our custom code.